client/js/actions/view-action-creator.js

```diff
@@ -11,12 +11,12 @@ var ViewActionCreator = {
     ApiUtils.addEvent(event, ServerActionCreator.eventAdded);
   },
 
-  getDayData: function(){
+  getDayData: function(date){
```

Comment:
At the moment we have separate utility functions for getting day, week, month data. Is this best practice or can we abstract them all into one function?